package.json

```diff
@@ -50,10 +51,15 @@
     "react": "^17.0.1",
     "react-beautiful-dnd": "^13.1.0",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.2.0",
+    "react-pro-sidebar": "^0.6.0",
     "react-responsive-carousel": "^3.2.19",
+    "react-slidedown": "^2.4.5",
+    "resize-observer-polyfill": "^1.5.1",
     "stimulus": "^1.1.1",
     "tailwindcss": "^2.0.1",
     "terser-webpack-plugin": "^1.4.5",
+    "tslib": "^1.14.1",
     "webpack": "^4.35.2",
     "webpack-assets-manifest": "^3.1.1",
     "webpack-cli": "^3.3.5",
```

Comment:
How come there are these changes in package.json? Have you tried external dependencies and forgot to remove them from this file when decided not to use them? :)

Reply:
Pleas remove before merge if they are not used in the code :)